spotinst_sdk2/clients/ocean/__init__.py

```diff
@@ -3,20 +3,24 @@
 from spotinst_sdk2.client import Client
 import spotinst_sdk2.models.ocean.aws as aws_ocean
 
+
 class OceanAwsClient(Client):
```

Comment:
All APIs - account_id need not be passed explicitly. It is retrieved from the client object.

Reply:
Done, checked